src/main/java/spireMapOverhaul/zones/goldencurse/GoldenCurseZone.java

```diff
@@ -0,0 +1,114 @@
+package spireMapOverhaul.zones.goldencurse;
+
+import com.badlogic.gdx.graphics.Color;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.ImageMaster;
+import com.megacrit.cardcrawl.rewards.RewardItem;
+import spireMapOverhaul.abstracts.AbstractZone;
+import spireMapOverhaul.patches.CustomRewardTypes;
+import spireMapOverhaul.rewards.SingleCardReward;
+import spireMapOverhaul.util.ActUtil;
+import spireMapOverhaul.zoneInterfaces.RewardModifyingZone;
+
+import java.util.ArrayList;
+
+import static spireMapOverhaul.patches.CustomRewardTypes.SMO_SINGLECARDREWARD;
+
+public class GoldenCurseZone extends AbstractZone implements RewardModifyingZone {
+    public static final String ID = "GoldenCurse";
+
+    public static final int BASE_CARD_COST = 30;
+    public static final int BASE_HEAL_COST = 10;
+
+    private final Color color = Color.GOLD.cpy();
+
+    public GoldenCurseZone() {
+        super(ID);
+        this.width = 2;
+        this.maxWidth = 3;
+        this.height = 2;
+        this.maxHeight = 3;
+        iconTexture = ImageMaster.MAP_NODE_ELITE;
```

Comment:
Also, maybe use the golden idol relic as image? Would be pretty fitting

Reply:
will do, I totally forgot to change it.